packages/shared/src/components/modals/OnboardingModal.tsx

```diff
@@ -127,6 +138,15 @@ function OnboardingModal({
     }
   };
 
+  useEffect(() => {
+    trackEvent({
+      event_name: 'impression',
+      target_type: 'onboarding modal',
+      target_id: 'v2',
+      extra: JSON.stringify({ origin: mode }),
+    });
```

Comment:
This is not inline with the proposed taxonomy:

https://docs.google.com/spreadsheets/d/18Lv7zXges9QfVX5VYL1a-Hyl0e1sQ3sLr0OK8YZWKXI/edit#gid=0

(My feed onboarding)

Reply:
Thank you for pointing this out, I have now a better understanding of the taxonomy sheet.